vmdb/spec/models/ems_refresh/refreshers/k8s_refresher_spec.rb

```diff
@@ -39,42 +40,41 @@ def assert_ems
   def assert_specific_container
     @container = Container.find_by_name("heapster")
     @container.should have_attributes(
-      :ems_ref       => "49984e80-e1b7-11e4-b7dc-001a4a5f4a02_heapster_kubernetes/heapster:v0.9",
+      # :ems_ref       => "a7566742-e73f-11e4-b613-001a4a5f4a02_heapster_kubernetes/heapster:v0.9",
```

Comment:
@blomquisg How did you solve the problem of the ids constantly changing for the other providers?  I can't remember.